### ISSUE TITLE
very low validation performance with pretrained model

### ISSUE THREAD

**Issue by sharkovsky** [FETS-AI/Models#1]
Hi, 

I am trying to reproduce the results from this [paper](https://www.nature.com/articles/s41467-022-33407-5), by extracting (i.e. copy-pasting) the pytorch code as-is from the various repos in the FeTS-AI org, as needed. It is relevant to do this, instead of directly installing openfl, to ensure reproducibility across different frameworks (I plan to extend to other FL frameworks that support pytorch when this issue is solved).

The paper states that, using the 'Public Initial Model', an average DSC score of roughly 0.8 was achieved, corresponding to a DSC loss value of 0.2.
However, using the pretrained model provided in this repo, I get very high loss values (around 0.9), corresponding to a DSC score of 0.1! 

You can find what I did [here](https://gist.github.com/sharkovsky/f5af051862422b43bfd8c5362bdf62c8), and if you scroll to the bottom you'll see the bad loss values. 

I understand that there may be some slight differences in performance, but not by such a large factor. Maybe the provided model is not the one referenced in the paper?

Thank you! 
Francesco

**Comment by sarthakpati** [comment]
Hi @sharkovsky,

Thank you for your interest in the FeTS study! 

The model you are using is actually just a reference model that we had created to act as an initialization step. The actual consensus models are [here](https://github.com/FeTS-AI/Front-End#consensus-models-for-2021-study).

I would like to reiterate that we are not maintaining this code for anything related to AI, and we are using GaNDLF/OpenFL for all our work [[ref](https://github.com/FeTS-AI/Algorithms/issues/83#issuecomment-1655578462)]. To replicate our study, please use the installer [[ref](https://github.com/FeTS-AI/Front-End#downloads)] or the Docker image [[ref](https://github.com/FeTS-AI/Front-End/pkgs/container/front-end)].

Cheers,
Sarthak

**Comment by sharkovsky** [comment]
Thank you @sarthakpati!

It's all clear, I had missed the fact that the consensus models were available for download in the readme, sorry! 

Thank you for clarifying the fact that you're not maintaining the code, I wasn't sure where to post the issue so apologies if I made a mistake. Thanks in any case for your clear answer, I will test the consensus model immediately! 

I'm very interested in the FeTS study, thank you for your responsiveness.

**Comment by sarthakpati** [comment]
Sure thing, no worries at all! 

To give some historical context, the FeTS study was designed to be a pilot study to showcase the efficacy of FL in a large-scale real-world setup, and all the effort was done to put that together. Once the study concluded, we realized that doing any further research on it would be tough (because we had designed it for 1 **_very specific_** use case), so we spun off the different elements to make feature addition and maintenance easier. 

As far as the FeTS tool itself is concerned, we are going to provide barebones support to ensure the consensus model can be inferred via either the available installer or the docker image, and any further research will be done on GaNDLF/OpenFL.

Hope this clarifies things, and please don't hesitate to contact the team either here or via email (you can find it on my profile page). 

Cheers and all the best!
